R/commonAnovaBayesian.R

```diff
@@ -1799,6 +1800,145 @@ BANOVAcomputMatchedInclusion <- function(effectNames, effects.matrix, interactio
+    errorBarType  <- options$errorBarTypeTwo
+    confInterval <- options$confidenceIntervalIntervalTwo
+    descriptivesPlotTwoContainer$dependOn(c("dependent", "plotTwoErrorBars", "errorBarTypeTwo", "zeroFix",
+                                            "confidenceIntervalIntervalTwo", "usePooledStandErrorCITwo"))
+
+  }
+  usePooledSE <- if (is.null(options[["usePooledStandErrorCITwo"]])) FALSE else options[["usePooledStandErrorCITwo"]]
+  zeroFix <- options$zeroFix
+  descriptivesPlotTwoContainer$dependOn(c("plotTwoHorizontalAxis", "plotTwoSeparatePlots", "labelYAxisTwo"))
+
+  if (errors$noVariables) {
+    descriptivesPlotTwoContainer[["dummyplot"]] <- createJaspPlot(title = gettext("Bar Plot"))
+    return()
+  }
+
+  groupVars <- c(options$plotTwoHorizontalAxis, options$plotTwoSeparatePlots)
+  groupVars <- groupVars[groupVars != ""]
+  if (analysisType == "RM-ANOVA") {
+    dependent <- .BANOVAdependentName
+    yLabel <- options[["labelYAxisTwo"]]
+  } else {
+    dependent<- options$dependent
+    yLabel <- options[["dependent"]]
+  }
+
+  betweenSubjectFactors <- groupVars[groupVars %in% options$betweenSubjectFactors]
+  repeatedMeasuresFactors <- groupVars[groupVars %in% sapply(options$repeatedMeasuresFactors, `[[`, "name")]
+
+  if (length(repeatedMeasuresFactors) == 0) {
+    summaryStat <- jaspTTests::summarySE(as.data.frame(dataset), measurevar = dependent, groupvars = groupVars,
+                                         conf.interval = confInterval, na.rm = TRUE, .drop = FALSE,
+                                         errorBarType = errorBarType, dependentName = .BANOVAdependentName,
+                                         subjectName = .BANOVAsubjectName)
+  } else {
+    summaryStat <- jaspTTests::summarySEwithin(as.data.frame(dataset), measurevar= .BANOVAdependentName,
+                                               betweenvars = betweenSubjectFactors,
+                                               withinvars = repeatedMeasuresFactors,
+                                               idvar = .BANOVAsubjectName,
+                                               conf.interval = confInterval,
+                                               na.rm=TRUE, .drop = FALSE, errorBarType = errorBarType,
+                                               usePooledSE = usePooledSE, dependentName = .BANOVAdependentName,
+                                               subjectName = .BANOVAsubjectName)
+  }
+
+  colnames(summaryStat)[colnames(summaryStat) == dependent] <- "dependent"
+
+  if (options$plotTwoHorizontalAxis != "") {
+    colnames(summaryStat)[colnames(summaryStat) == options$plotTwoHorizontalAxis] <- "plotTwoHorizontalAxis"
+  }
+
+  if (options$plotTwoSeparatePlots != "") {
+    colnames(summaryStat)[colnames(summaryStat) == options$plotTwoSeparatePlots] <- "plotTwoSeparatePlots"
+    subsetPlots <- levels(summaryStat[,"plotTwoSeparatePlots"])
+    nPlots <- length(subsetPlots)
+  } else {
+    nPlots <- 1
```

Comment:
```suggestion
    nPlots <- 1L
```
since you use `1L` in other places, it's best to just be consistent and assign an integer everywhere.